doc/lsst.ip.diffim/tasks/lsst.ip.diffim.DecorrelateALKernalSpatialTask.rst

```diff
@@ -0,0 +1,28 @@
+.. lsst-task-topic:: lsst.ip.diffim.DecorrelateALKernelSpatialTask
+
+##########################
+DecorrelateALKernelSpatialTask
+##########################
+
+Pipe-task that removes the neighboring-pixel covariance in an
+image difference that are added when the template image is
+convolved with the Alard-Lupton PSF matching kernel.
+
+.. _lsst.ip.diffim.DecorrelateALKernelSpatialTask-description:
+
+Description
+==================
+
+This task is a simple wrapper around @ref DecorrelateALKernelTask,
```

Comment:
These `@ref THING` should be changed into RST links. So, this one would be `~lsst.ip.diffim.imageDecorrelation.DecorrelateALKernelTask`, I think. It should show up as an actual link when correctly formatted.